flytekit/extras/pytorch/__init__.py

```diff
@@ -0,0 +1,20 @@
+"""
+Flytekit PyTorch
+=========================================
+.. currentmodule:: flytekit.extras.pytorch
+
+.. autosummary::
+   :template: custom.rst
+   :toctree: generated/
+
+    PyTorchCheckpoint
+"""
+from flytekit.loggers import logger
+
+try:
+    from .checkpoint import PyTorchCheckpoint, PyTorchCheckpointTransformer
```

Comment:
we should stick with the full import in the future, just for consistency.  merge as is, i'll update it in the future.